maintenance/wikia/getDatabase.php

```diff
@@ -193,26 +193,36 @@ function getFile($databaseDirectory, $dbname, $filedir) {
 		require_once( $dirName . "/../../../config/DB.php" );
 	}
 
-	if ( isset( $wgDBbackenduser, $wgDBbackendpassword, $wgLBFactoryConf['hostsByName']['sharedb-s4'] ) ) {
+	if ( isset( $wgDBbackenduser, $wgDBbackendpassword ) ) {
 		// prepare raw output for consumption as csv. changes " => \"; \t => ","; beginning of line => ", end of line => "
 		$prepareCsv = "sed 's/\"/\\\\\"/g;s/\\t/\",\"/g;s/^/\"/;s/$/\"/;s/\\n//g'";
 
-		$dbhost = $wgLBFactoryConf['hostsByName']['sharedb-s4'];
+		// This is kind of a hack
+		$dbhost = 'slave.db-sharedb.service.sjc.consul';
 	    // dump city_list row to local CSV file and import into local database
 		$response = `mysql -u $wgDBbackenduser -p$wgDBbackendpassword --database wikicities -h $dbhost -ss -e "SELECT * from city_list where city_id = $city_id " | $prepareCsv > /tmp/city_list.csv`;
 		print "city_list dump ok\n";
 		$response = `mysqlimport -u $wgDBdevboxUser -p$wgDBdevboxPass -h $wgDBdevboxCentral --replace --fields-enclosed-by=\\" --fields-terminated-by=, --local wikicities /tmp/city_list.csv`;
 		print $response;
 		unlink ("/tmp/city_list.csv");
+
 		// dump city_domains rows to local CSV file and improt into local database
 		$response = `mysql -u $wgDBbackenduser -p$wgDBbackendpassword --database wikicities -h $dbhost -ss -e "SELECT * from city_domains where city_id = $city_id " | $prepareCsv > /tmp/city_domains.csv`;
 		print "city_domains dump ok\n";
 		$response = `mysqlimport -u $wgDBdevboxUser -p$wgDBdevboxPass -h $wgDBdevboxCentral --replace --fields-enclosed-by=\\" --fields-terminated-by=, --local wikicities /tmp/city_domains.csv`;
 		print $response;
 		unlink ("/tmp/city_domains.csv");
-		// dump city_vars rows to local CVS file and import into local database
+
+		// dump city_variables_pool rows to local CVS file and import into local database
+		$response = `mysql -u $wgDBbackenduser -p$wgDBbackendpassword --database wikicities -h $dbhost -ss -e "select * from city_variables_pool where cv_id in (select cv_variable_id from city_variables where cv_city_id = $city_id)" | $prepareCsv > /tmp/city_variables_pool.csv`;
+		print "city_variables_pool dump ok\n";
+		$response = `mysqlimport -u $wgDBdevboxUser -p$wgDBdevboxPass -h $wgDBdevboxCentral --replace --fields-enclosed-by=\\" --fields-terminated-by=, --local wikicities /tmp/city_variables_pool.csv`;
+		print $response;
+		unlink ("/tmp/city_variables_pool.csv");
```

Comment:
I think it's a good moment to remove this repeatable dump and import sections and use a function to perform them.